### PR TITLE
Add set_cover_tilt_position translation to Polish

### PR DESCRIPTION
**Pull request by kstaniek**
This is just Polish translation for set_cover_tilt_position action

### CODE REVIEW

File: src/localize/languages/pl.json [hunk 1]
```diff
@@ -15,6 +15,8 @@
       "close_cover": "zamknij",
       "open_cover": "otwórz",
       "set_cover_position": "ustaw pozycję[ na {position}]"
```
Comment (KTibow) [inline src/localize/languages/pl.json:17]: The JSON is invalid.
```suggestion
      "set_cover_position": "ustaw pozycję[ na {position}]",
```

**Comment by nielsfaber** [comment]
@KTibow thanks for reviewing.
Do you know of any way to have these translation files validated on commit (using GitHub action or something)?
Ideally I would like to check if the file has the same JSON entries as the EN file, so check for completeness.
Seems like these translation files are very difficult to maintain up-to-date.

**Comment by KTibow** [comment]
You could whip up a Python script using the json and glob libraries to go through each json file and try to load it. Then just run it via GitHub actions.

File: src/localize/languages/pl.json (hunk 1 above)
Comment (nielsfaber) [inline src/localize/languages/pl.json]:
```suggestion
      "set_cover_position": "ustaw pozycję[ na {position}]",

```
```

File: src/localize/languages/pl.json [hunk 2]
```diff
@@ -14,7 +14,9 @@
     "cover": {
       "close_cover": "zamknij",
       "open_cover": "otwórz",
-      "set_cover_position": "ustaw pozycję[ na {position}]"
+      "set_cover_position": "ustaw pozycję[ na {position}]",
+      "set_cover_tilt_position": "ustaw pozycję lameli[ na {tilt_position}]"
+    },
```
Comment (nielsfaber) [inline src/localize/languages/pl.json:19]:
```suggestion
```

**Pull request merged**